modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/impl/TestMvPartitionStorage.java

```diff
@@ -271,6 +271,12 @@ private VersionChain resolveCommittedVersionChain(VersionChain committedVersionC
             // Calling it from the compute is fine. Concurrent writes of the same row are impossible, and if we call the compute closure
             // several times, the same tuple will be inserted into the GC queue (timestamp and rowId don't change in this case).
             gcQueue.add(committedVersionChain);
+        } else {
+            if (committedVersionChain.row == null) {
+                // If previous version doesn't exist and current version is a tombstone,
```

Comment:
I would paraphrase in: 
`if only one version, and it is a tombstone, then we will remove such a chain.`